packages/fxa-auth-server/scripts/delete-inactive-accounts/get-inactive-account-uids.ts

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env node -r esbuild-register
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A script to get a list of inactive account UIDs.
+ *
+ * For expediency, this script has very few parameters.  It will rely on the
+ * same set of enivronment variables as the auth server.
+ */
```

Comment:
I love all the documentation on the parameters but I'd still like to see an example up here.  Like, "here's what it looks like to get all accounts which haven't had activity in the past 2 years"

Reply:
Added https://github.com/mozilla/fxa/pull/17899/files#diff-61eb2369e7ed3635471be49d4975dfab988f5b76af07c7f7f4adfb075d7c38e3R57-R63